Doc/library/stdtypes.rst

```diff
@@ -4254,7 +4254,10 @@ pairs within braces, for example: ``{'jack': 4098, 'sjoerd': 4127}`` or ``{4098:
       Create a new dictionary with keys from *iterable* and values set to *value*.
 
       :meth:`fromkeys` is a class method that returns a new dictionary. *value*
-      defaults to ``None``.
+      defaults to ``None``.  The same instance of *value* is used for every
```

Comment:
Since value is not a class, 'instance of value' does not make sense.
```suggestion
      defaults to ``None``.  The one *value* object is used for every
```